lib/service/Service.ts

```diff
@@ -425,8 +425,8 @@ class Service extends EventEmitter {
   /**
    * resolveHash resolve hash to preimage.
    */
-  public resolveHash = async (request: lndrpc.ResolveRequest) => {
-    return this.swaps.resolveHash(request);
+  public resolveHash = async (request: ResolveRequest) => {
```

Comment:
At the time of working on this PR, I thought we might still be using this for Raiden. Since then it seems that we won't, and I'm removing obsolete code/references like this in #940.